MITADS/opensubtitles_exporter.py

```diff
@@ -121,22 +122,19 @@ def parsexmlfile(xml_path, count_file):
 count_file = 0
 
 # Parse 5 files at once
-pool = ThreadPoolExecutor(max_workers=20)
+pool = ProcessPoolExecutor(max_workers=20)
```

Comment:
It's not worth specifying the number of workers manually, better to let ProcessPoolExecutor use its default: the number of processors on the machine.